src/v/config/configuration.cc

```diff
@@ -480,6 +480,26 @@ configuration::configuration()
        .visibility = visibility::user},
       model::timestamp_type::create_time,
       {model::timestamp_type::create_time, model::timestamp_type::append_time})
+  , log_message_timestamp_alert_before_ms(
+      *this,
+      "log_message_timestamp_alert_before_ms",
+      "Threshold in milliseconds for alerting on messages with a timestamp "
+      "after the broker's time, meaning they are in the future relative to the "
```

Comment:
This is the past setting :)

You'll probably want to s/future/past and s/after/before here and update the below description.